### PR TITLE
Art and design subject page

### PR DESCRIPTION
**Pull request by mariosh-e**
### Trello card

https://trello.com/c/xggRhYAy

### CODE REVIEW

File: app/views/content/life-as-a-teacher/art-and-design.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Become an art and design teacher
+title_paragraph: |-
+  <p>
+  Teaching art and design opens up a world of creativity for students.  When students create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/art-and-design.md:5]:
```suggestion
  Teaching art and design opens up a world of creativity for pupils.  When pupils create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
```

File: app/views/content/life-as-a-teacher/art-and-design/_article.html.erb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
+      link_title: "You can get help comparing English and international qualifications.",
+      link_url: "/non-uk-teachers/non-uk-qualifications",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_artanddesign %> are available for eligible trainee art and design teachers.</p>
+    <p>You can get a bursary alongside a tuition fee loan and maintenance loan.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/art-and-design/_article.html.erb:68]: what about scholarships?

File: app/views/content/life-as-a-teacher/art-and-design/_article.html.erb [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      title: "check your qualifications",
+      text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
+      link_title: "You can get help comparing English and international qualifications.",
+      link_url: "/non-uk-teachers/non-uk-qualifications",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_artanddesign %> are available for eligible trainee art and design teachers.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/art-and-design/_article.html.erb:67]: Need to make sure the full stop isn't included in the hyperlink?

File: app/views/content/life-as-a-teacher/art-and-design/_article.html.erb (hunk 2 above)
Comment (mariosh-e) [reply app/views/content/life-as-a-teacher/art-and-design/_article.html.erb:68]: No scholarships for Art and design

File: app/views/content/life-as-a-teacher/art-and-design/_article.html.erb [hunk 4]
```diff
@@ -0,0 +1,82 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach art and design</h2>
+    <div class="inset">
+    <p>Teaching art and design gives students the technical skills needed to express their ideas in a wide variety of materials and media. You'll help students develop their own skills by encouraging them to critically review their work. Your students will develop an appreciation of art and design, and of the diverse ideas and perspectives of others.</p>
```
[the remark this answers is not in the document]
Comment (mariosh-e) [reply app/views/content/life-as-a-teacher/art-and-design/_article.html.erb]: @Sarah-DfE

File: app/views/content/life-as-a-teacher/art-and-design.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+title: Become an art and design teacher
+title_paragraph: |-
+  <p>
+  Teaching art and design opens up a world of creativity.  When pupils create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_artanddesign$ are available for eligible trainee art and design teachers.</p>
+description: |-
+    Find out how to become an art and design teacher, including what you'll be teaching and what funding is available to help you train.
+subcategory: What to teach
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/art-and-design.md:10]: This needs to be removed, as this page should be in the Explore subjects section now

File: app/views/content/life-as-a-teacher/art-and-design/_article.html.erb [hunk 6]
```diff
@@ -0,0 +1,108 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach art and design</h2>
+     <div class="inset">
+    <p>Teaching art and design gives pupils the technical skills needed to express their ideas in a wide variety of materials and media. You'll help pupils develop their own skills by encouraging them to critically review their work. Your pupils will develop an appreciation of art and design, and of the diverse ideas and perspectives of others.</p>
+    <p>In addition to the personal rewards of teaching art and design, there are practical benefits too, including:</p>
+      <ul>
+        <li>a <a href="/life-as-a-teacher/pay-and-benefits">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/art-and-design/_article.html.erb:8]:
```suggestion
        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
```

File: app/views/content/life-as-a-teacher/art-and-design/_article.html.erb [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+    <%= render Content::QuoteComponent.new(
+    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+    name: "Chris Farr, teacher training adviser",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Teaching art and design opens the door for your pupils to pursue exciting careers in fields as diverse as fashion and textiles, to illustration and video game design.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/art-and-design/_article.html.erb:26]:
```suggestion
      <p>Teaching art and design opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from fashion and textiles, to illustration and video game design.</p>
```
Suggestion for readability but up to you!

File: app/views/content/life-as-a-teacher/art-and-design/_article.html.erb [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+      <p>By thinking creatively, your pupils could make significant contributions to these diverse and impactful areas.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/art-and-design/_article.html.erb:27]:
```suggestion
      <p>By thinking creatively, your pupils could make significant contributions to these impactful areas.</p>
```
Felt like 'diverse' was being repeated a lot

File: app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb [hunk 9]
```diff
@@ -3,6 +3,12 @@
     <section class="container category__cards main-section">
       <nav class="category__nav-cards">
         <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Art and design",
+              description: "Find out how to become an art and design teacher and inspire children to fuel their passion for creativity.",
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb:9]:
```suggestion
              description: "Become an art and design teacher and inspire pupils to fuel their passion for creativity.",
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
+---
+title: Become an art and design teacher
+title_paragraph: |-
+  <p>
+  Teaching art and design opens up a world of creativity.  When pupils create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md:5]:
```suggestion
  Teaching art and design opens up a world of creativity. When pupils create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_artanddesign$ are available for eligible trainee art and design teachers.</p>
+description: |-
+    Find out how to become an art and design teacher, including what you'll be teaching and what funding is available to help you train.
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md:9]:
```suggestion
    Explore what's involved in becoming an art and design teacher, including what you'll be teaching and what funding is available to help you train.
```

**Review by Sarah-DfE — COMMENTED** [review]
tweaks to nav card and front matter

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 12]
```diff
@@ -0,0 +1,108 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach art and design</h2>
+     <div class="inset">
+    <p>Teaching art and design gives pupils the technical skills needed to express their ideas in a wide variety of materials and media. You'll help pupils develop their own skills by encouraging them to critically review their work. Your pupils will develop an appreciation of art and design, and of the diverse ideas and perspectives of others.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:5]:
```suggestion
    <p>Teaching art and design gives pupils the technical skills needed to express their ideas through a range of materials and forms of media. You'll help pupils develop their own skills by encouraging them to critically review their work. Your pupils will develop an appreciation of art and design, and of the diverse ideas and perspectives of others.</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+    <p>In addition to the personal rewards of teaching art and design, there are practical benefits too, including:</p>
+      <ul>
+        <li>a <a href="/life-as-a-teacher/pay-and-benefits">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:8]:
```suggestion
        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+    <%= render Content::QuoteComponent.new(
+    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+    name: "Chris Farr, teacher training adviser",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Teaching art and design opens the door for your pupils to pursue exciting careers in fields as diverse as fashion and textiles, to illustration and video game design.</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:26]:
```suggestion
      <p>Teaching art and design opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from fashion and textiles, to illustration and video game design.</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+      <li>observation and recording techniques</li>
+      <li>media exploration</li>
+      <li>proficiency in material handling</li>
+      <li>creativity and idea development</li>
+      <li>contextual understanding</li>
+    </ul>
+  </section>
+
+  <section class="col col-720">
+    <p>In key stage 4 (14 to 16 years olds) you will help pupils demonstrate skills in:</p>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:45]:
```suggestion
    <p>In key stage 4 (14 to 16 years olds) you'll help pupils demonstrate skills in:</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop a deeper knowledge and understanding of:</p>
+    <ul>
+      <li>how ideas, feelings and meanings can be conveyed and interpreted in images and artefacts</li>
+      <li>continuity and change in different genres, styles and traditions</li>
+      <li>recording experiences and observations, in a variety of ways</li>
+      <li>exploring relevant resources; analyse, discuss and evaluate images, objects and artefacts; and make and record independent judgements</li>
+      <li>using knowledge and understanding of the work of others to develop and extend thinking and inform their own work</li>
+      <li>generate and explore potential lines of enquiry using appropriate media and techniques</li>
+  </section>
+
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:62]: <p>Teaching pupils at different ages and abilities means you'll always be expanding your knowledge and approach to meet pupils' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
</section>

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+  <section class="col col-720">
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop a deeper knowledge and understanding of:</p>
+    <ul>
+      <li>how ideas, feelings and meanings can be conveyed and interpreted in images and artefacts</li>
+      <li>continuity and change in different genres, styles and traditions</li>
+      <li>recording experiences and observations, in a variety of ways</li>
+      <li>exploring relevant resources; analyse, discuss and evaluate images, objects and artefacts; and make and record independent judgements</li>
+      <li>using knowledge and understanding of the work of others to develop and extend thinking and inform their own work</li>
+      <li>generate and explore potential lines of enquiry using appropriate media and techniques</li>
+  </section>
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:61]:
```suggestion
```

**Review by Sarah-DfE — COMMENTED** [review]
Adding sections that match the complete template

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+title_paragraph: |-
+  <p>
+  Teaching art and design opens up a world of creativity. When pupils create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_artanddesign$ are available for eligible trainee art and design teachers.</p>
+description: |-
+    Explore what's involved in becoming an art and design teacher, including what you'll be teaching and what funding is available to help you train.
+layout: "layouts/minimal"
+colour: pastel yellow-yellow
+image: "static/images/content/hero-images/maths.jpg"
```
Comment (Sarah-DfE) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md:12]:
```suggestion
image: "static/images/content/hero-images/0031.jpg"
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md [hunk 19]
```diff
@@ -0,0 +1,22 @@
+---
+title: Become an art and design teacher
+title_paragraph: |-
+  <p>
+  Teaching art and design opens up a world of creativity. When pupils create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_artanddesign$ are available for eligible trainee art and design teachers.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design.md:7]:
```suggestion
  Tax-free bursaries are available for eligible trainee art and design teachers.</p>
```
Remove bursaries value as figure is from last year

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Teaching art and design opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from fashion and textiles, to illustration and video game design.</p>
+      <p>By thinking creatively, your pupils could make significant contributions to these diverse and impactful areas.</p>
+      <p>Your pupils will learn more from you than you might think.  As their teacher, you'll guide them though the knowledge and skills they need to show their understanding of art and design. You'll teach them valuable life skills such as critical thinking, adaptability and communication.</p>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-art-and-design-programmes-of-study/national-curriculum-in-england-art-and-design-programmes-of-study">national curriculum for art and design</a>.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:32]: Should this open in the same tab? Or is that on a future ticket?

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 21]
```diff
@@ -0,0 +1,109 @@
...
+    <ul>
+      <li>using a range of techniques to record observations in sketchbooks, journals and other media as a basis for exploring ideas</li>
+      <li>using range of techniques and media, including painting</li>
+      <li>increasing proficiency in the handling of different materials</li>
+      <li>teaching pupils to analyse and evaluate their own work, and that of others, in order to strengthen the visual impact or applications of their work</li>
+      <li>the history of art, craft, design and architecture, including periods, styles and major movements from ancient times up to the present day</li>
+    </ul>
+    <p><a href="https://www.thenational.academy/teachers/programmes/art-secondary-ks3-l/units">Explore what teaching art and design would be like at key stage 3 with resources from Oak National Academy<span class="visually-hidden">(opens in new window)</span></a>.</p>
+
+    <p>In key stage 4 (14 to 16 years olds) you'll help pupils demonstrate skills in:</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:43]:
```suggestion
    <p>In key stage 4 (14 to 16 years olds), you'll help pupils demonstrate skills in:</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb [hunk 22]
```diff
@@ -0,0 +1,109 @@
...
+      title: "check your qualifications",
+      text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
+      link_title: "You can get help comparing English and international qualifications.",
+      link_url: "/non-uk-teachers/non-uk-qualifications",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_artanddesign %> are available for eligible trainee art and design teachers</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:83]:
```suggestion
    <%= render Content::InsetTextComponent.new(text: "The figures on this page apply if you're starting your training $bursaries_fundingdates$. Financial support for courses starting the following year will be announced in autumn 2024.") %>
    <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_artanddesign %> are available for eligible trainee art and design teachers</p>
```
Need to add holding message inset text

**Review by emmaachesongray — COMMENTED** [review]
Couple of comments for you :)

File: app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb (hunk 20 above)
Comment (Sarah-DfE) [reply app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb:32]: @emmaachesongray future ticket, Jen can update all the links at once I think

**Review by emmaachesongray — APPROVED** [review]
Looks good to me 👍

**Pull request merged**